### DISCUSSION TITLE
Add Icon Color as Attribute

### DISCUSSION THREAD

**Discussion by tegner23** [kalkih/mini-graph-card#1133] [Ideas]
Is there a possibility to set up a specific color for the icon?

I use the graph cards for energy consumption and I would like to have a switch icon which changes color depending on the state of the switch (on or off). Therefor an attribute to set the icon color would be appricated.

![image](https://github.com/user-attachments/assets/5d999039-19b2-4c26-82c3-819147869980)

**Comment by ildar170975** [comment]
Ask in the main card-mod thread of Community.
